beef_tx_sorting.go

```diff
@@ -57,9 +59,9 @@ func getTxWithZeroIncomingEdges(incomingEdgesMap map[string]int) []string {
 	return zeroIncomingEdgeQueue
 }
 
-func removeTxFromIncomingEdges(tx *Transaction, incomingEdgesMap map[string]int, zeroIncomingEdgeQueue []string) []string {
-	for _, input := range tx.draftTransaction.Configuration.Inputs {
-		neighborID := input.UtxoPointer.TransactionID
+func removeTxFromIncomingEdges(tx *bt.Tx, incomingEdgesMap map[string]int, zeroIncomingEdgeQueue []string) []string {
+	for _, input := range tx.Inputs {
+		neighborID := input.PreviousTxIDStr() // TODO: perf
```

Comment:
On what measures we assume that performance optimisation is needed?

Reply:
I added a more detailed comment